src/Components/blinds.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import Open from '../Images/blinds-open.svg';
+import Closed from '../Images/blinds-closed.svg';
+import './blinds.css';
+
+const Blinds = ({ state }) => {
+    const [isOpen, setIsOpen] = useState(state === "open")
+
+    const handleClick = () => {
+        setIsOpen(!isOpen);
+    }
+
+    return (
+        <div className="blinds">
+            <div className="blinds__icon">
+                <img src={isOpen ? Open : Closed} alt={isOpen ? "zaluzie otevreny" : "zaluzie zatazeny"}/>
+            </div>
+            <div className="blinds__name">
+                Žaluzie
+            </div>
+            <div className="blinds__controls">
+                <button className={isOpen ? "button button--active" : "button"} disabled={isOpen} onClick={handleClick}>Otevřeno</button>
+                <button className={!isOpen ? "button button--active" : "button"} disabled={!isOpen} onClick={handleClick}>Zavřeno</button>
```

Comment:
kdyby sis to podmínečné generování classy chtěla zjednodušit, tak https://www.npmjs.com/package/classnames